src/js/listeners/auth/register.js

```diff
@@ -1,24 +1,45 @@
-import * as auth from "../../api/auth/index.js";
+import * as auth from '../../api/auth/index.js';
 
 export async function registerListener(event) {
-  event.preventDefault()
-  const form = event.target
-  const data = new FormData(form)
-  const email = data.get("email")
-  const name = data.get("name")
-  const password = data.get("password")
-  const avatar = data.get("avatar")
+  event.preventDefault();
+  const form = event.target;
+  const data = new FormData(form);
 
-  try {
-    await auth.register(name, email, password, avatar)
-  } catch {
-    return alert("There was a problem creating your account")
+  const email = data.get('email');
+  if (!email.endsWith('@stud.noroff.no') && !email.endsWith('@noroff.no')) {
```

Comment:
Again, the task is not to change the original code. If you do not feel like the code is adequate, report this in the issues tab. It is not your responsibility to fix or improve the code, only to test it.